Documentation/internals/hooks.rst

```diff
@@ -0,0 +1,16 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+eBPF Program Types
+==================
```

Comment:
Suggestion: Make it more explicit from ToC that we're not documenting the types?

```suggestion
eBPF Program Types in Use
=========================
```

Reply:
OK yeah, the text below it is probably good enough indeed